Resources/Prototypes/_RedEclipse/Recipes/Factory/Assemblers/assembler_medical.yml

```diff
@@ -1,42 +1,34 @@
 #Factory Medical Assembler
-- type: latheRecipe
-  id: FactoryMacroBombImplanter
-  result: MacroBombImplanter
-  completetime: 1
-  materials: # TODO better recipe
-    FactoryBasicParts : 1
-
 - type: latheRecipe
   id: FactoryStorageImplanter
   result: StorageImplanter
   completetime: 1
   materials: # TODO better recipe
-    FactoryBasicParts : 1
-
-- type: latheRecipe
-  id: FactoryFreedomImplanter
-  result: FreedomImplanter
-  completetime: 1
-  materials: # TODO better recipe
-    FactoryBasicParts : 1
+    FactoryBasicCasings : 1
+    FactoryInjectors: 1
 
 - type: latheRecipe
   id: FactoryHypopenBox
   result: HypopenBox
   completetime: 1
   materials: # TODO better recipe
-    FactoryBasicParts : 1
+    FactoryBasicCasings : 1
+    FactoryInjectors: 1
+    FactoryMedicalParts: 1
 
 - type: latheRecipe
-  id: FactorySyndicateSmokesPacket
-  result: CigPackSyndicate
-  completetime: 1
```

Comment:
Well, if you remove FactorySyndicateSmokesPacket proper commit would be "Removed FactorySyndicateSmokesPacket recipe", bro, it's impossible to figure all this out properly fast when proper commits or changelog are non-existent